### ISSUE TITLE
[BUG] Extensions not working

### ISSUE THREAD

**Issue by nico-abram** [Microsoft/vscode#63376]
Issue Type: Bug

I first realized formatting wasnt working in my lua files (Using https://marketplace.visualstudio.com/items?itemName=trixnz.vscode-lua ), even when *only* using that extension. Trying ctrl+shift+p and Format Document told me i had no formatter for lua files.

After doing "Toggle Developer Tools" I found 2 errors. I'm relatively sure all extensions don't work for me for the same reason. Here are some logs:

Using ONLY vscode-lua : 
[-1542526776712.log](https://github.com/Microsoft/vscode/files/2592577/-1542526776712.log)

Using ONLY gitlen : 
[-1542526826942.log](https://github.com/Microsoft/vscode/files/2592578/-1542526826942.log)

Using ONLY Prettier : 
[-1542526853215.log](https://github.com/Microsoft/vscode/files/2592579/-1542526853215.log)

My OS is Windows 7 Ultimate 64bit. Everything worked fine, today for some reason it stopped working. Also, I tried updating using the in-program updater, but i kept getting a "Path not found error". I figured my install might be wacky so I uninstalled and manually installed the latest version. Still getting the same error. I can't think of anything strange I might have done to make it stop working (Except maybe installing visual studio 2017 and rustc).

Any help is appreciated. I have also tried running as administrator, but I got the exact same errors.

**Comment by nico-abram** [comment]
Just tried running the zip vscode in portable mode (I created a data folder in the folder containing code.exe) with no extensions and got the same errors, in case that helps/means anything.

**Comment by roblourens** [comment]
```
extensionService.ts:153 Error: spawn UNKNOWN
    at _errnoException (util.js:1024:11)
    at ChildProcess.spawn (internal/child_process.js:323:11)
    at exports.spawn (child_process.js:514:9)
    at Object.exports.fork (child_process.js:115:10)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3168:318
    at Object.g [as _notify] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:162:971)
```

and the same error when trying to start the filewatcher. I see some other issues with a similar error but nothing that was really resolved.

I don't know where to go with this one, @joaomoreno have you seen this before?

**Comment by nico-abram** [comment]
I managed to fix it. It seems my windows profile was somehow corrupted. A *lot* of profile specific things weren't working profile, including setting some important env vars like appdata, windir, etc. After a couple hours i didn't manage to fix it (Not by making a new profile, not by messing with the registry, not by running a tool called reprofiler, not by messing with use groups in a windows tool), so I just manually added most env vars i could think. It works now. My guess is this was related to that? Not sure if it's worth fixing
